### ISSUE TITLE
How to set image styles in local web UI?

### ISSUE THREAD

**Issue by ericzhangcn1980** [InstantID/InstantID#160]
I have successfully deployed the Instand ID project in web UI. However, the local Web UI does not have the various style options like in official demo, such as style1-8 or 'Spring Festival', 'Watercolor', 'Film Noir', 'Neon', 'Jungle', 'Mars', 'Vibrant Color', etc. Are these styles implemented through prompts? THX

**Comment by haofanwang** [comment]
Yes, you can find these prompt templates [here](https://github.com/ahgsql/StyleSelectorXL/blob/main/sdxl_styles.json).

**Comment by ericzhangcn1980** [comment]
> Yes, you can find these prompt templates [here](https://github.com/ahgsql/StyleSelectorXL/blob/main/sdxl_styles.json).

THX, it's very valuable!

**Comment by rocing** [comment]
THX
